deployment/deployment.json

```diff
@@ -2731,6 +2731,28 @@
           }
         }
       },
+      "evm-base": {
```

Comment:
Do you want to call this `evm-base-testnet`?

Reply:
Maybe we just leave base out until it is mainnet. I don't see much of a point to have a testnet deployment rn. I will make it a draft again